pre-docs/running_bolt_from_docker_image.md

```diff
@@ -0,0 +1,182 @@
+# Puppet-bolt container
+
+Puppet Bolt it available as a docker image. Container images are published to [dockerhub](https://hub.docker.com/r/puppet/puppet-bolt/tags) with tags corresponding to bolt system package and Rubygem versions in addition to a `latest` tag which points at the newest version. This document covers some possible ways to use the container.
+
+## Downloading the image
+
+You can download the latest image from dockerhub with the following command:
```

Comment:
```suggestion
You can download the latest image from Docker Hub with the following command:
```

Reply:
If downloading the latest image is something that the user has to do (i.e., it's not an optional step), you can make this a lot shorter:
`Download the latest image from Docker Hub:`

If it is optional, you can still shorten it somewhat:
`You can download the latest image from Docker Hub:`